src/media/inner.rs

```diff
@@ -424,17 +438,16 @@ impl MediaInner {
         now: Instant,
         exts: &ExtensionMap,
         twcc: &mut u64,
-        pad_size: Option<usize>,
         buf: &mut Vec<u8>,
-    ) -> Option<(RtpHeader, SeqNo)> {
+    ) -> Option<(RtpHeader, SeqNo, bool, usize)> {
```

Comment:
Maybe it's time we have a name typed returned here rather than a tuple with four elements.

```rust
struct PolledPacket {
  header: RtpHeader,
  twcc_seq_no: SeqNo,
  is_padding: bool,
  payload_size: usize,
}
```